### PR TITLE
Added storage account changes in make.json for VsTestV2

### PR DESCRIPTION
**Pull request by virekumar**
**Task name**: VsTestV2

**Description**: Changed location of external resources present in make.json

**Documentation changes required:** Not required

**Added unit tests:** Not required

**Attached related issue:** No

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

**Testing**

image

image

### CODE REVIEW

File: Tasks/VsTestV2/Tests/package-lock.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@types/mocha": {
       "version": "5.2.5",
-      "resolved": "https://pkgs.dev.azure.com/mseng/PipelineTools/_packaging/PipelineTools_PublicPackages/npm/registry/@types/mocha/-/mocha-5.2.5.tgz",
```
Comment (adityashahms) [inline Tasks/VsTestV2/Tests/package-lock.json:8]: Any particular reason to move away from our secured feed to the public one?

**Review by adityashahms — COMMENTED** [review]
Rest looks good to me

File: Tasks/VsTestV2/Tests/package-lock.json (hunk 1 above)
Comment (virekumar) [reply Tasks/VsTestV2/Tests/package-lock.json:8]: These got auto added on build

Comment (adityashahms) [reply Tasks/VsTestV2/Tests/package-lock.json:8]: Searching is rest of the code base, I see couple of places where official one is used. I guess it should be fine.

**Review by adityashahms — APPROVED** [review]
LGTM

**Pull request merged**